cmd/oras/internal/option/remote.go

```diff
@@ -47,6 +47,7 @@ type Remote struct {
 	Username          string
 	PasswordFromStdin bool
 	Password          string
+	IdentityToken     string
```

Comment:
Instead of having `IdentityToken`, we can have something like `Secret` and `SecretFromStdin`.

Reply:
Oh, So should I add a field secret that will hold the Identity Token, Instead of Password?